tests/providers/openai/test_openai.py

```diff
@@ -0,0 +1,39 @@
+from typing import Tuple
+
+import os
+import pytest
+
+from exchange import Text
+from exchange.message import Message
+from exchange.providers.base import Usage
+from exchange.providers.openai import OpenAiProvider
+from .conftest import OPENAI_MODEL, OPENAI_API_KEY
+
+
+@pytest.mark.vcr()
+def test_openai_completion(monkeypatch):
+    # When running VCR tests the first time, it needs OPENAI_API_KEY to call
```

Comment:
Took me some to understand this, maybe something like this is clearer (if too long, the key point is calling out real vs fake key)

```
When running VCR tests for the first time or after deleting a cassette recording, a real OPENAI_API_KEY must be passed as an environment variable so real responses can be fetched.
Subsequent runs use the recorded data so we allow running without any environment variables by setting a fake key value.
```